src/policies/single_policy_functions.py

```diff
@@ -111,12 +113,19 @@ def _process_multiplier(states, multiplier, name):
     if isinstance(multiplier, (pd.Series, pd.DataFrame)):
         date = get_date(states)
         multiplier = multiplier.loc[date]
-    msg = f"Work {name} multiplier not in [0, 1] on {get_date(states)}"
+    neg_multiplier_msg = f"Work {name} multiplier < 0 on {get_date(states)}"
+    larger_one_msg = (
+        f"Work {name} multiplier > 1. For recurrent models it is not possible to "
```

Comment:
I think I would rather handle this in a data management step and issue one warning. This will probably produce several warnings per simulated day.

Reply:
This is a great suggestion. Done.